extensions/amp-analytics/0.1/config.js

```diff
@@ -66,9 +66,32 @@ export class AnalyticsConfig {
 
     return this.fetchRemoteConfig_()
       .then(this.processConfigs_.bind(this))
+      .then(this.handleTopLevelAttributes_.bind(this))
```

Comment:
It'll be too late to handle `requestOrigin` here. We need to handle that before the config merge. Because it could be that some analytics vendor provides
```
'testRequest': {
  'baseUrl': 'test',
  'requestOrigin': 'vendorRequestOrigin'
}
``` 

Then the inline config is 
```
{
  'requestOrigin': 'inlineRequestOrigin'
}
```

We want the inline configuration to override the vendor configuration. And that requires us to handle top level attributes for every config before we merge them. 
`#expandConfigRequest` might be a nice place